openflow13/match.go

```diff
@@ -873,6 +874,39 @@ func NewMplsBosField(mplsBos uint8) *MatchField {
 	return f
 }
 
+// MplsTc field
+type MplsTcField struct {
+	MplsTc uint8
+}
+
+func (m *MplsTcField) Len() uint16 {
+	return 1
+}
+
+func (m *MplsTcField) MarshalBinary() (data []byte, err error) {
+	data = make([]byte, 1)
+	data[0] = m.MplsTc
+	return
+}
+func (m *MplsTcField) UnmarshalBinary(data []byte) error {
+	m.MplsTc = data[0]
```

Comment:
Please check length of `data` before accessing data[0]

Reply:
thanks, could you please review it again? Thanks !